packages/metrics/modules/create-metric-types/create-metric-types.js

```diff
@@ -24,15 +24,22 @@ const defaultGcLabels = ['gc_type'];
 const areMetricsInSecondsEnabled = options => options.accuracies.includes('s');
 const areMetricsInMillisecondsEnabled = options =>
   options.accuracies.includes('ms');
-const areSummariesEnabled = options => options.metricTypes.includes('summary');
+const areSummariesEnabled = options =>
+  options.metricTypes.includes('httpRequestsSummary');
 const areHistogramsEnabled = options =>
-  options.metricTypes.includes('histogram');
+  options.metricTypes.includes('httpRequestsHistogram');
+const areRequestsTotalEnabled = options =>
```

Comment:
Note to self: rename these to `shouldObserveRequestsInSummary`, `shouldObserveRequestsInHistogram` and `shouldObserveRequestsInCounter`.